workflow/hosts/orion.yaml

```diff
@@ -1,12 +1,12 @@
-BASE_GIT: '/work/noaa/global/glopara/git_rocky9'
-DMPDIR: '/work/noaa/rstprod/dump'
-BASE_DATA: '/work/noaa/global/glopara/data'
-BASE_IC: '/work/noaa/global/glopara/data/ICSDIR'
-PACKAGEROOT: '/work/noaa/global/glopara/nwpara'
-COMINsyn: '/work/noaa/global/glopara/com/gfs/prod/syndat'
-HOMEDIR: '/work2/noaa/global/${USER}/ORION'
-STMP: '/work2/noaa/global/${USER}/ORION/STMP'
-PTMP: '/work2/noaa/global/${USER}/ORION/PTMP'
+BASE_GIT: '/work2/noaa/global/role-global/git'
+DMPDIR: '/lfs/h2/emc/da/noscrub/catherine.thomas/obs/combined'
```

Comment:
This DMPDIR should be the same as hercules: /work2/noaa/da/cthomas/obs/combined'

Reply:
i swear i changed this like 3 times...

Reply:
updated